Makefile

```diff
@@ -91,6 +91,9 @@ SINGLE_MAKE = export MAKEFLAGS= ; $(MAKE)
 nvim: build/.ran-cmake deps
 	+$(BUILD_TOOL) -C build
 
+asan:
+	$(MAKEFILE_DIR)/contrib/asan.sh
```

Comment:
Why do we need a makefile target that points to a script? And, the next question will be a familiar one about cmake...

Reply:
Sure, `contrib` sounds good